src/applications/mhv-landing-page/tests/containers/AppConfig.unit.spec.jsx

```diff
@@ -24,14 +24,14 @@ describe(`${appName} -- <AppConfig />`, () => {
     const useRumSpy = sinon.spy(useDatadogRum);
     const initialState = initialStateFn();
 
-    const { findByText } = renderWithStoreAndRouter(
+    const { getByText } = renderWithStoreAndRouter(
       <AppConfig useDatadogRumFn={useRumSpy}>
         <p>child node</p>
       </AppConfig>,
       { initialState },
     );
 
-    findByText('child node');
+    getByText('child node');
```

Comment:
Note: `findBy` returns a promise and this use will always pass since the promise is not checked with await.